packages/sample/src/main.ts

```diff
@@ -5,7 +5,9 @@ import {RevHub} from "@rev-robotics/expansion-hub";
 const program = new Command();
 
 program.version('1.0.0')
-    .option('-l --list', 'List connected devices').parse(process.argv);
+    .option('-l --list', 'List connected devices')
+    .option('-c --close', 'Test closing a hub')
```

Comment:
I'd argue that the sample should always end by closing any hubs it opened, and then we don't need to add a separate option to test closing.

Reply:
Especially because the sample should show best practices for how to use the library.

Reply:
The list task currently closes all hubs when done (lets them keep-alive for 2 seconds first). This makes the close task redundant, I suppose. Removed it.